### PR TITLE
Use alias for astroid.nodes 02

### PR DESCRIPTION
**Pull request by cdce8p**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :hammer: Refactoring   |

## Description

Followup to #4656

### CODE REVIEW

File: pylint/checkers/strings.py [hunk 1]
```diff
@@ -213,13 +214,13 @@
 }
 
 OTHER_NODES = (
```
Comment (Pierre-Sassoulas) [inline pylint/checkers/strings.py:216]: Not related to this MR but this variable name is hard to understand.

**Review by Pierre-Sassoulas — APPROVED** [review]
Nice work ! Just a nitpick we could remove docstrings and replace them by typing in signature when we have to update docstrings.

**Comment by cdce8p** [comment]
> Nice work ! Just a nitpick we could remove docstrings and replace them by typing in signature when we have to update docstrings.

Doing so would enable type checking in that function which often results in mypy errors. I think it would be better if we do that another time.

**Comment by Pierre-Sassoulas** [comment]
You're right.

**Pull request merged**